roles/mesos/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 # Common
 consul_dir: /etc/consul.d
 mesos_executor_registration_timeout: 10mins
-mesos_cluster_name: "Cluster01"
+mesos_cluster_name: "{{ mesos_cluster_name }}"
```

Comment:
i thought it was odd we were setting this explicitly here when we have the global var